components/RadioButtons.tsx

```diff
@@ -33,7 +33,7 @@ const RadioButtons = ({
 
   return (
     <>
-      {title && <h3>{title}</h3>}
+      {title && <h3 className={styles.title}>{title}</h3>}
```

Comment:
The other questions now have centered question titles, but this one isn't centered. Do we want to move those centering styles to the global h2/h3 style instead?